okio/jvm/src/main/resources/META-INF/proguard/okio.pro

```diff
@@ -0,0 +1,5 @@
+# JSR 305 annotations are for embedding nullability information.
+-dontwarn javax.annotation.**
```

Comment:
These aren't used in Okio anymore right?

Reply:
I just checked the dependencies and saw it there https://github.com/square/okio/blob/master/okio/jvm/build.gradle#L55

Reply:
GitHub search doesn't find any usages though so I'll remove it.